04-reference/environment-variables.mdx

```diff
@@ -24,10 +24,38 @@ description: "This are the environment variables for AskUI"
 - **ASKUI_INFERENCE_ENDPOINT**: The URL of the Vision Agent's inference endpoint, which handles image and text analysis.
 - **ASKUI_CONTROLLER_PATH**: The file path to the controller's executable or binary file.
 - **ASKUI__VA__TELEMETRY__ENABLED**: A boolean flag to enable or disable the recording of usage data for the Vision Agent. Setting this variable to False will disable telemetry.
+- **ASKUI_CONTROLLER_ARGS**: Arguments to pass to the AskUI Remote Device Controller executable. This variable accepts a string of command-line arguments that control the behavior of the controller.
+
+  **Supported arguments:**
+  - `--showOverlay [true|false]`: Controls whether the controller shows an overlay on the screen during automation. Default is `true`.
+  - `--debugDraw [true|false]`: Enables or disables debug drawing features for troubleshooting.
+  - `--configFile <AbsolutePathToConfigFile>`: Specifies the path to a configuration file for the controller.
+
+  **Examples:**
+  ```bash
+  # Disable overlay and use custom config file
+  ASKUI_CONTROLLER_ARGS="--showOverlay false --configFile /path/to/config.json"
+  
+  # Disable overlay only
+  ASKUI_CONTROLLER_ARGS="--showOverlay false"
+  
+  # Enable debug drawing
+  ASKUI_CONTROLLER_ARGS="--debugDraw true"
+  ```
+
+  **Default value:** `--showOverlay true`
 
 ---
 
 ### AskUI Suite:
 
 - **ASKUI_COMPONENT_REGISTRY_FILE**: The path to a file that defines custom UI components for AskUI.
 - **ASKUI_INSTALLATION_DIRECTORY**: The directory where the AskUI suite is installed.
+- **ASKUI_DISABLE_TELEMETRY**: A boolean flag to disable the recording of usage data for the AskUI suite. Setting this variable to True will disable telemetry.
+
+---
+
+### AskUI UI Controller Watchdog Windows Service:
```

Comment:
Please be consistent in naming. Take the name from the commandlet or change the name from the commandlet